src/common/config.cpp

```diff
@@ -56,4 +115,21 @@ void Config::loadModelParameters(const void* ptr) {
   override(config);
 }
 
+void Config::override(const YAML::Node& params) {
```

Comment:
Please add a comment to specify exactly how that override process happens. E.g. does it replace entire nodes (so that existing entries in those nodes get deleted if the new one does not have one of the same key), or is the result the union set, with parameters occuring in both set to the overriding one?